exercises/leap/canonical-data.json

```diff
@@ -1,24 +1,34 @@
 {
-   "cases": [
-      {
-         "description": "year not divisible by 4: common year",
-         "input": 2015,
-         "expected": false
-      },
-      {
-         "description": "year divisible by 4, not divisible by 100: leap year",
-         "input": 2016,
-         "expected": true
-      },
-      {
-         "description": "year divisible by 100, not divisible by 400: common year",
-         "input": 2100,
-         "expected": false
-      },
-      {
-         "description": "year divisible by 400: leap year",
-         "input": 2000,
-         "expected": true
-      }
-   ]
+  "exercise": "leap",
+  "version": "1.0.0",
+  "comments": [
+    "Given a Gregorian calendar year, indicate whether it is a leap year."
+  ],
+
```

Comment:
This comment was not in the original test suite and also is not needed for compliance, so maybe it should not be part of this commit/PR.